modules/administration/pages/disconnected-setup.adoc

```diff
@@ -122,6 +122,33 @@ You can change SMT user settings in [path]``/etc/smt.conf``.
 
 
 
+== Mandatory Channels
+
+The corresponding {productname} Client Tools Channels are required, for {productname} to be able to synchronize a given channel.
+If these channels are not enabled, {productname} may fail to detect that product.
+
+Run the following command to enable these mandatory channels:
+
+SLES 12::
```

Comment:
Not only SLES, but also all other products based on SLES, like SLES for SAP and HPC, etc.

Reply:
ok, thanks!